docs/interfaces/cli-ug.rst

```diff
@@ -0,0 +1,348 @@
+.. _cli-ug:
+
+################
+ Determined CLI
+################
+
+To use Determined, you'll need, at minimum, the Determined Command-Line Interface (CLI) and a
+Determined cluster. The Determined CLI includes the ``det`` command line tools for interacting with
+a Determined cluster. This page contains instructions for using the CLI, including installion and
+upgrade.
+
+.. warning::
+
+   Although Determined supports password-based authentication, communication between the Determined
+   CLI, Determined WebUI, and the Determined master does not take place over an encrypted channel by
+   default.
+
+.. note::
+
+   All users should install the Determined CLI on their local development machine.
+
+.. note::
+
+   You can also interact with Determined using the :ref:`web-ui-if`.
+
+.. _install-cli:
+
+**************
+ Installation
+**************
+
+The CLI is distributed as a Python wheel package. The CLI requires Python >= 3.7. For best results,
```

Comment:
nit: think this would benefit from shortening up. Something like

The CLI is distributed as a Python wheel package and requires Python >= 3.7. We recommend installing the CLI into a [virtualenv](https://virtualenv.pypa.io/en/latest/).

Reply:
more concise

Reply:
resolved

Reply:
@garrett361 all issues are resolved, ready for re-review